File: CS/ConditionalAppearanceExample.Module.Web/Controllers/WebConditionalAppearanceController.cs
using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Diagnostics;
using System.Text;

using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.Base;
using ConditionalAppearanceExample.Module.Controllers;
using DevExpress.ExpressApp.ConditionalAppearance;
using DevExpress.ExpressApp.Web.Editors;
using DevExpress.ExpressApp.Web.Editors.ASPx;
using System.Web.UI.WebControls;
using System.Drawing;

namespace ConditionalAppearanceExample.Module.Web.Controllers {
   public partial class WebConditionalAppearanceController : ConditionalAppearanceController {

      protected override void CustomizeDisabledEditorsAppearance(ApplyAppearanceEventArgs e) {
         base.CustomizeDisabledEditorsAppearance(e);
         WebPropertyEditor dxEditor = e.Item as WebPropertyEditor;
         if (dxEditor != null && dxEditor.Editor != null) {
            dxEditor.Editor.BorderStyle = BorderStyle.Dashed;
            dxEditor.Editor.BackColor = Color.RosyBrown;
         }
      }
   }
}

File: VB/ConditionalAppearanceExample.Module.Web/Controllers/WebConditionalAppearanceController.vb
Imports System
Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Text

Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.Base
Imports ConditionalAppearanceExample.Module.Controllers
Imports DevExpress.ExpressApp.ConditionalAppearance
Imports DevExpress.ExpressApp.Web.Editors
Imports DevExpress.ExpressApp.Web.Editors.ASPx
Imports System.Web.UI.WebControls
Imports System.Drawing

Namespace ConditionalAppearanceExample.Module.Web.Controllers
   Partial Public Class WebConditionalAppearanceController
       Inherits ConditionalAppearanceController

      Protected Overrides Sub CustomizeDisabledEditorsAppearance(ByVal e As ApplyAppearanceEventArgs)
         MyBase.CustomizeDisabledEditorsAppearance(e)
         Dim dxEditor As WebPropertyEditor = TryCast(e.Item, WebPropertyEditor)
         If dxEditor IsNot Nothing AndAlso dxEditor.Editor IsNot Nothing Then
            dxEditor.Editor.BorderStyle = BorderStyle.Dashed
            dxEditor.Editor.BackColor = Color.RosyBrown
         End If
      End Sub
   End Class
End Namespace
